src/scenes/AcceleratorRouter/ParticipantProjects/Reports/NewProjectSpecificMetric.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { Container, Grid, Typography, useTheme } from '@mui/material';
+import { Dropdown } from '@terraware/web-components';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+
+import Page from 'src/components/Page';
+import Card from 'src/components/common/Card';
+import Checkbox from 'src/components/common/Checkbox';
+import PageForm from 'src/components/common/PageForm';
+import TextField from 'src/components/common/Textfield/Textfield';
+import { APP_PATHS } from 'src/constants';
+import { selectCreateProjectMetric } from 'src/redux/features/reports/reportsSelectors';
+import { requestCreateProjectMetric, requestProjectReportConfig } from 'src/redux/features/reports/reportsThunks';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import strings from 'src/strings';
+import { NewMetric } from 'src/types/AcceleratorReport';
+import useForm from 'src/utils/useForm';
+
+export default function NewProjectSpecificMetric(): JSX.Element {
+  const theme = useTheme();
+  const navigate = useNavigate();
+  const pathParams = useParams<{ projectId: string }>();
+  const projectId = Number(pathParams.projectId);
```

Comment:
I'm curious why we're converting `projectId` to a number if we convert it back to a string in several of its usages? And if the answer is just "because we always convert numeric strings to numbers then that makes sense to me haha

Reply:
projectId is a number in the BE, so if we needed to "work" with it, it would be a number. In this case I'm just using it in the api url and we need it as a string only, so you are right, it would be simpler to just use it as a string!